CONTRIBUTING.md

```diff
@@ -384,39 +396,58 @@ Here is a skeleton predicate script:
 # * we access any undefined variable.
 set -eu
 
-# Print out Rust backtraces on panic. Useful for minimizing a particular panic.
-export RUST_BACKTRACE=1
+# Invoke the general purpose predicate script that comes in the
+# `bindgen` repository.
+path/to/rust-bindgen/csmith-fuzzing/predicate.py \
+    $PREDICATE_FLAGS \
+    ./isolated-test-case.h
+```
+
+When hunting down a particular panic emanating from inside `bindgen`, you can
+invoke `predicate.py` like this:
 
-# If the `libclang.so` you're using for `bindgen` isn't the system
-# `libclang.so`, let the linker find it.
-export LD_LIBRARY_PATH=~/path/to/your/directory/containing/libclang
+```bash
+path/to/rust-bindgen/csmith-fuzzing/predicate.py \
+    --expect-bindgen-fail \
+    --bindgen-grep "thread main panicked at '<insert panic message here>'" \
+    ./isolated-test-case.h
+```
 
-# Make sure that the reduced test case is valid C/C++ by compiling it. If it
-# isn't valid C/C++, this command will exit with a nonzero exit code and cause
-# the whole script to do the same.
-clang[++ --std=c++14] -c ./pre_processed_header.hpp
```

Comment:
`predicate.py` doesn't do this anymore and `predicate.sh` doesn't contain this either. 
Is it useful?

Reply:
Ultimately, I don't think so.